introduction.md

```diff
@@ -1,2 +1,8 @@
 Greetings fellow deveopers! I'm Ryan and I've returned to finally finish this course!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?